src/DonationForms/FormDesigns/TwoPanelStepsFormLayout/css/_sections.scss

```diff
@@ -0,0 +1,16 @@
+.givewp-donation-form__steps {
+    @media screen and (max-width: 71.5rem) {
```

Comment:
Where does this `71.5rem` value come from?

Reply:
That's the space necessary for the two columns plus extra margins:
`0.5rem (form pad) + 35rem (col1) + 0.5rem (gap) + 35rem (col2) + 0.5rem (form pad) = 71.5rem`

Reply:
oh interesting, so is that basically your calculation for when to break for tablet?

Reply:
Exactly. If we don't have space enough for the two columns, so break it in a single one.